### DISCUSSION TITLE
Forward Nodes failed to Upgrade after ManagerSearch got upgraded

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#7806] [Unsupported Versions]
I upgraded ManagerSearch Node from 2.3.90->2.3.110 by running soup, the forward nodes did you not get upgraded. And after I rebooted one of the forward nodes, all the services are failing in that forward node. I get the following error:

 so-zeek-restart --force
=========================================================================
Restarting zeek...

This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================

Force-stopping all Salt jobs before proceeding

local:
Error response from daemon: No such container: so-zeek
Error: No such container: so-zeek
[ERROR   ] Failure!
local:
----------
          ID: zeek_state_not_allowed
    Function: test.fail_without_changes
      Result: False
     Comment: Failure!
     Started: 11:48:10.717663
    Duration: 2.096 ms
     Changes:

Summary for local
------------
Succeeded: 0
Failed:    1
------------
Total states run:     1
Total run time:   2.096 ms

Any suggestions and help would be greatly appreciated.

**Comment by dougburks** [comment]
On the manager, what is the output of `sudo salt \* test.ping`?

On the forward node, what is the output of `sudo salt-call state.highstate`?
